charts/refarch-templates/README.md

```
@@ -133,15 +133,15 @@ Example:
 #### Base information
 
 Base information involves setting the module name and the docker image (and tag) to use.
-Additionally, you can override the `pullPolicy`, which is set to `IfNotPresent` by default.
+Additionally, you can override the `pullPolicy`, which is set to `Always` by default.
 
 Example:
 ```yaml
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)